lib/ReactViews/FeatureInfo/FeatureInfoSection.tsx

```diff
@@ -396,7 +396,8 @@ export class FeatureInfoSection extends React.Component<FeatureInfoProps> {
     if (this.props.catalogItem.featureInfoTemplate.name) {
       title = Mustache.render(
         this.props.catalogItem.featureInfoTemplate.name,
-        this.featureProperties
+        this.mustacheContextData,
+        this.props.catalogItem.featureInfoTemplate.partials
```

Comment:
Need to pass all the template parameters like the other call [above](https://github.com/TerriaJS/terriajs/blob/5cc4472cd87196bb80e9317513e547f03ce6cc96/lib/ReactViews/FeatureInfo/FeatureInfoSection.tsx#L96-L100).